lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -179,6 +179,23 @@ def compare_results(
                 {},
                 ["test", "fio_1", "fio_2", "uperf_1", "uperf_2", "uperf_3", "uperf_4"],
             ),
+            ("drb", {"owner": "drb"}, ["drb", "fio_1"]),
+            ("drb", {"mine": True}, ["drb", "fio_1"]),
+            ("drb", {"mine": "t"}, ["drb", "fio_1"]),
+            ("drb", {"mine": "T"}, ["drb", "fio_1"]),
+            ("drb", {"mine": "true"}, ["drb", "fio_1"]),
+            ("drb", {"mine": "True"}, ["drb", "fio_1"]),
+            ("drb", {"mine": "y"}, ["drb", "fio_1"]),
+            ("drb", {"mine": "yes"}, ["drb", "fio_1"]),
+            ("drb", {"mine": "Yes"}, ["drb", "fio_1"]),
```

Comment:
These cases all seem redundant (as evidenced by the fact that they all have the same `results` value).  That is, what you're really testing with all these cases is `convert_boolean()`, so it seems like this parametr-set belongs on `test_boolean()` in `lib/pbench/test/unit/server/test_schema.py` rather than here.  (Ditto for the `false` cases below.)

Reply:
I was more interested in getting the coverage; and when you pushed to cover the redundant cases, I just added them here. Strictly speaking, yeah, it could have been done in the `Parameter` tests either instead or in addition (because making sure it works at this level isn't necessarily "wrong" either). Sigh.